packages/fxa-auth-server/lib/senders/emails/layouts/fxa/index.ts

```diff
@@ -17,7 +17,10 @@ export const render = (body: string) => {
       </mj-title>      
     </mj-head>
     <mj-body css-class="body-80">
-      <mj-raw>
+    <mj-include path="./lib/senders/emails/css/global.css" type="css" css-inline="inline" />
+    <mj-include path="./lib/senders/emails/css/fxa/index.css" type="css" css-inline="inline" />
+    <mj-include path="./lib/senders/emails/css/rtl.css" type="css" />
```

Comment:
I had to create a separate file for rtl and ltr classes since we don't want these styles to be inlined. These styles are getting applied dynamically so either we do this way or render the styles from inside Js something like this:
```
if(isLocaleRenderedRtl){
      const head = document.getElementsByTagName('head')[0];
      const style = document.createElement("style");
      /* @ts-ignore */
      // style.type = 'text/css'
      style.innerHTML = `.rtl div {         
        direction: rtl !important;
       }`;
      head.appendChild(style);
    }
```